CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Add `cargo contract encode` command - [#998](https://github.com/paritytech/cargo-contract/pull/998)
+- Add `info` command - [#993](https://github.com/paritytech/cargo-contract/pull/993)
```

Comment:
Please merge `master`into your PR, you'll have to put this into the unreleased section again then.